src/validators/json-schema.coffee

```diff
@@ -90,14 +127,25 @@ class JsonSchema
 
     results = []
 
-    #amanda to gavel converter
+    #TV4 to gavel converter
     if data.length > 0
       indexes = [0..data.length - 1]
       indexes.forEach (index) ->
         item = data[index]
-        pointer = item['property'] or []
+
+        if item['property'] == undefined
+          pathArray = []
+        else if item['property'] == null
+          pathArray = []
+        else if item['property'] == [null]
```

Comment:
You cannot do this type of comparison, it won't work in JS.
Please do `Array.isArray(item['property']) and item['property'].length is 1 and item['property'][0] is null`

Or simply we can remove these checks, they don't seem useable to me.
Can you please provide some way they can actually happen?